pinax/stripe/actions/customers.py

```diff
@@ -45,12 +45,15 @@ def create(user, card=None, plan=settings.PINAX_STRIPE_DEFAULT_PLAN, charge_imme
     """
     trial_end = hooks.hookset.trial_period(user, plan)
 
-    stripe_customer = stripe.Customer.create(
-        email=user.email,
-        source=card,
-        plan=plan,
-        trial_end=trial_end
-    )
+    kwargs = {
+        'email': user.email,
+        'source': card,
+        'plan': plan,
+        'trial_end': trial_end,
+    }
+    if coupon:
+        kwargs.update({'coupon': coupon})
```

Comment:
This is similar to a [change I've got in my PR](https://github.com/pinax/pinax-stripe/pull/319/files#diff-b46b4ff78f8c8f8e951ee3a56eac38a9R30).

Perhaps we'd both benefit from all `kwargs` from this function being passed into `stripe.Customer.create`...